_targets.R

```diff
@@ -0,0 +1,38 @@
+library(targets)
+source("data_extraction_ms.R")
+tar_option_set(packages = c("tidyverse", "sbtools", "whisker","readr","dplyr","stringr"))
+list(
+ # define global vars   
+   tar_target(file_name, file_name <- 'model_RMSEs.csv'),
+   tar_target(sb_id_sciencebase,sb_id_sciencebase <- '5d925066e4b0c4f70d0d0599'),
+   tar_target(downloaded_data_folder, downloaded_data_folder <- '1_fetch/out'),
```

Comment:
@jesse-ross I am not clear on what this bot wants to see changed.

Reply:
I think maybe it wants you to use the full path to the file name as a single parameter, rather than compose it from a path and a file name. The reason is that the `_targets.R` should be easily readable by a human as well as by R. In a real collaborative workflow, you'd often want to glance at the source to see where a file ended up, so it's nice to see its full path all in one place.

So to make this more human-readable, I think you could change your `data_extraction()` function to just take the full path and name as a single argument like `1_fetch/out/model_RMSEs.csv`, rather than passing in `1_fetch/out` and `model_RMSes.csv` separately and joining them in the function as you're currently doing.